docs/operations/memory_hotplug.md

```diff
@@ -0,0 +1,136 @@
+# Memory Hotplug
+
+Memory hotplug was introduced in KubeVirt version 1.1, enabling the dynamic resizing of the amount of memory available to a running VM.
```

Comment:
It would be good to separate the feature description from the version it was introduced into separate sentences, possibly even paras. That way we won't have to rewrite it when the version is no longer relevant.
And this might possibly be a nit as maybe there's enough information but is there anything else we could add to the feature description for someone that might be completely new to memory hotplugging?

Reply:
It's a good suggestion but honestly, I can't think of anything else to add, memory hotplug is in general a well-understood concept in virtualization.